app/scripts/modules/core/src/entityTag/notifications/notifications.less

```diff
@@ -1,6 +1,8 @@
-@import (reference) "~core/presentation/less/imports/commonImports.less";
+@import (reference) '~core/presentation/less/imports/commonImports.less';
```

Comment:
I think the fancy post-commit hook fixed this up?

Reply:
Yeah, that is prettier doing its thing.